src/mercator/mercator/static/i18n/mercator_en.json

```diff
@@ -189,6 +189,13 @@
     "TR__PERSONAL_CONTACT": "Personal contact",
     "TR__PROPOSAL_ADD": "Add proposal",
     "TR__PROPOSAL_EMPTY_TEXT": "No proposals yet.",
-    "TR__TWITTER": "twitter",
-    "TR__USERS_PROPOSALS": "{name}'s proposals"
+    "TR__TWITTER": "Twitter",
+    "TR__USERS_PROPOSALS": "{name}'s proposals",
+    "TR__MERCATOR_FIRSTNAME": "First name",
+    "TR__MERCATOR_LASTNAME": "Last name",
+    "TR__MERCATOR_16_OTHER": "Other",
+    "TR__MERCATOR_16_WEBSITE": "Website",
+    "TR__MERCATOR_16_COUNTRY": "Country",
+    "TR__MERCATOR_16_ONLINE": "Online",
+    "TR__MERCATOR_16_NEWSLETTER": "Newsletter"
```

Comment:
you should use the extract_messages script instead of manually appending fields. This way the correct order is preserved.